python/test/unit/fem/test_assemble_submesh.py

```diff
@@ -119,86 +119,138 @@ def test_submesh_facet_assembly(n, k, space, ghost_mode):
 
 
 @pytest.mark.parametrize("n", [4, 6])
-@pytest.mark.parametrize("k", [1, 4])
+@pytest.mark.parametrize("k", [1, 3])
```

Comment:
What is the reason for this reduction?

Reply:
No reason. I think third degree tests the same things as fourth but it a bit cheaper. I can put it back though if you prefer

Reply:
Just worried that it was removing a failing test. no worries, don't need to add it back